libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -801,13 +817,14 @@ void AP_OSD_Screen::draw(void)
     DRAW_SETTING(home);
     DRAW_SETTING(roll_angle);
     DRAW_SETTING(pitch_angle);
+    DRAW_SETTING(flightime);
 
 #ifdef HAVE_AP_BLHELI_SUPPORT
     DRAW_SETTING(blh_temp);
     DRAW_SETTING(blh_rpm);
     DRAW_SETTING(blh_amps);
 #endif
-
+    DRAW_SETTING(message);
```

Comment:
Why message is moved? It will ovewrite other items now.

Reply:
Requested change by Kelly...I also prefer, but its a matter of personal preference..to be able to read messages if the horizon blocks them

Reply:
Make sense. But probably swap horizon and messages enouth.

Reply:
will do on next commit